src/cls/IPM/Utils/Module.cls

```diff
@@ -1213,6 +1266,11 @@ ClassMethod LoadNewModule(
     set tSC = $$$OK
     set tInitTLevel = $tlevel
     try {
+        // check module.xml for defaults
+        set tSC = ..GetModuleDefaultsFromXML(pDirectory, .defaults)
+        $$$ThrowOnError(tSC)
+        merge pParams = defaults
```

Comment:
This should be the other way around - currently a "default" will override a user-specified parameter.

To avoid module.xml defaults leaking back out byref, I'd recommend:
merge params = defaults
merge params = pParams

then use params through the rest of the method.